docs/resources/secret.md

```diff
@@ -0,0 +1,30 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "juju_secret Resource - terraform-provider-juju"
+subcategory: ""
+description: |-
+  A resource that represents a Juju secret.
+---
+
+# juju_secret (Resource)
```

Comment:
I wonder if we're going to have a merge conflict somewhere, you shouldn't be picking up this file. The original commit needs an example.tf too.